planner/core/plan_test.go

```diff
@@ -0,0 +1,171 @@
+package core_test
+
+import (
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb/domain"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/planner/core"
+	"github.com/pingcap/tidb/util/testkit"
+	"github.com/pingcap/tidb/util/testleak"
+)
+
+var _ = Suite(&testPlanNormalize{})
+
+type testPlanNormalize struct {
+	store kv.Storage
+	dom   *domain.Domain
+}
+
+func (s *testPlanNormalize) SetUpSuite(c *C) {
+	testleak.BeforeTest()
+	store, dom, err := newStoreWithBootstrap()
+	c.Assert(err, IsNil)
+	s.store = store
+	s.dom = dom
+}
+
+func (s *testPlanNormalize) TearDownSuite(c *C) {
+	s.dom.Close()
+	s.store.Close()
+	testleak.AfterTest(c)()
+}
+
+func (s *testPlanNormalize) TestNormalizedPlan(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t1,t2")
+	tk.MustExec("create table t1 (a int key,b int,c int, index (b));")
+	tk.MustExec("create table t2 (a int key,b int,c int, index (b));")
+	normalizedtCases := []struct {
```

Comment:
Please take a look at `testutil` package. don't write test in `_test.go` directly.

Reply:
Greate, done.